front/app/containers/Admin/projects/project/projectHeader/ReviewFlow/index.tsx

```diff
@@ -57,41 +46,33 @@ const ReviewFlow = ({ project }: { project: IProjectData }) => {
   }
 
   const approvalPending = projectReview?.data.attributes.state === 'pending';
-  const approvalGranted = projectReview?.data.attributes.state === 'approved';
-
-  const showProjectApprovalButton = approvalPending && canReview;
-  const showReviewRequestButton =
-    isProjectReviewEnabled && !canReview && !approvalGranted;
-  const showPublishButton =
-    !showProjectApprovalButton && !showReviewRequestButton;
+  const showReviewRequestButton = isProjectReviewEnabled && !canPublish;
+  const showProjectApprovalButton =
+    isProjectReviewEnabled && canPublish && approvalPending;
```

Comment:
"can publish" is functionally equivalent to "can review" here: when approval is pending it's basically an "is admin" or "can moderate the folder (if proj is in folder)" check.

Reply:
I'm not a huge fan of conflating the two. It happens to work with the current implementation, but mostly "by chance". I think the two concepts are semantically different enough to deserve their own permission rules. As a moderator, the fact that a project has been approved does not give you the permission to review it.